custom_components/waste_collection_schedule/waste_collection_schedule/source/impactapps_com_au.py

```diff
@@ -0,0 +1,321 @@
+        "url": "https://bmcc-waste.waste-info.com.au",
+        "website": "https://www.bmcc.nsw.gov.au",
+    },
+    {
+        "name": "Burwood City Council",
+        "url": "https://burwood-waste.waste-info.com.au",
+        "website": "https://www.burwood.nsw.gov.au",
+    },
+    {
+        "name": "Cowra Council",
+        "url": "https://cowra.waste-info.com.au",
+        "website": "https://www.cowracouncil.com.au/",
+    },
+    {
+        "name": "Forbes Shire Council",
+        "url": "https://forbes.waste-info.com.au",
+        "website": "https://www.forbes.nsw.gov.au",
+    },
+    {
+        "name": "Gwydir Shire Council",
+        "url": "https://gwydir.waste-info.com.au",
+        "website": "https://www.gwydir.nsw.gov.au",
+    },
+    {
+        "name": "Lithgow City Council",
+        "url": "https://lithgow.waste-info.com.au",
+        "website": "https://www.lithgow.nsw.gov.au",
+    },
+    {
+        "name": "Livingstone Shire Council",
+        "url": "https://livingstone.waste-info.com.au",
+        "website": "https://www.livingstone.qld.gov.au",
+    },
+    {
+        "name": "Loddon Shire Council",
+        "url": "https://loddon.waste-info.com.au",
+        "website": "https://www.loddon.vic.gov.au",
+    },
+    {
+        "name": "Moira Shire Council",
+        "url": "https://moira.waste-info.com.au",
+        "website": "https://www.moira.vic.gov.au",
+    },
+    {
+        "name": "Moree Plains Shire Council",
+        "url": "https://moree-waste.waste-info.com.au",
+        "website": "https://www.mpsc.nsw.gov.au",
+    },
+    {
+        "name": "Penrith City Council",
+        "url": "https://penrith.waste-info.com.au",
+        "website": "https://www.penrithcity.nsw.gov.au",
+    },
+    {
+        "name": "Port Macquarie Hastings Council",
+        "url": "https://pmhc.waste-info.com.au",
+        "website": "https://www.pmhc.nsw.gov.au",
+    },
+    {
+        "name": "Queanbeyan-Palerang Regional Council",
+        "url": "https://qprc.waste-info.com.au",
+        "website": "https://www.qprc.nsw.gov.au",
+    },
+    {
+        "name": "Singleton Council",
+        "url": "https://singleton.waste-info.com.au",
+        "website": "https://www.singleton.nsw.gov.au",
+    },
+    {
+        "name": "Snowy Valleys Council",
+        "url": "https://snowy-valleys.waste-info.com.au",
+        "website": "https://www.snowyvalleys.nsw.gov.au",
+    },
+    {
+        "name": "South Burnett Regional Council",
+        "url": "https://sbrc.waste-info.com.au",
+        "website": "https://www.southburnett.qld.gov.au",
+    },
+    {
+        "name": "Wellington Shire Council",
+        "url": "https://wellington.waste-info.com.au",
+        "website": "https://www.wellington.vic.gov.au",
+    }
+]
+SERVICE_MAP_LOOKUP = {council['name']: council for council in SERVICE_MAP}
+
+
+def EXTRA_INFO():
+    return [{"title": council['name'], "url": council['website']} for council in SERVICE_MAP]
```

Comment:
You could already add the "default_params": {"service": council['name']} which will be added for the GUI configuration (config_flow) in #2068. This will automatically fill in the service field when selecting a one of the EXTRA_INFO cases in the configuration form.

Reply:
This is a neat feature for this project. I've added the 'default_params' to the extra info.